sympy/ntheory/modular.py

```diff
@@ -250,3 +250,52 @@ def combine(c1, c2):
         if symmetric:
             return symmetric_residue(n, m), m
         return n, m
+
+def mod_inverse(a, m):
+    """
+    Given two number a  and m.This function computes the number
+    c such that   (a*c)%m==1.Meaning that if a is multiplied by c
+    then the remainder obtained on division with m is 1.
+    The value of m need not be a prime.This function  returns
+    the value of c  if some c exists.Otherwise it raises an
+    exception stating that modular inverse does not exist.
```

Comment:
I know I am being picky but I would also improve your white spaces: For example in your first sentence you have two white spaces between "a _ _ and" and then after the sentence no space at all between "m.This". It just looks cleaner if you always have exactly one white space ;)